optas/models.py

```diff
@@ -1818,12 +1818,154 @@ def get_global_link_axis_function(
     ) -> cs.Function:
         """! Get function for computing the link axis in the global frame.
 
-@param link Name of the end-effector link.
-@param axis The axis (direction vector) defined in the end-effector frame. If 'x', 'y', 'z' is passed then the corresponding sub-array of the homogenous transform is used.
-@param numpy_output When true, the output will be a NumPy array.
-@return A CasADi function that computes the link axis for a given joint state. If n > 1 then an array is computed whose columns correspond to the respective joint state in the input.
+        @param link Name of the end-effector link.
+        @param axis The axis (direction vector) defined in the end-effector frame. If 'x', 'y', 'z' is passed then the corresponding sub-array of the homogenous transform is used.
+        @param numpy_output When true, the output will be a NumPy array.
+        @return A CasADi function that computes the link axis for a given joint state. If n > 1 then an array is computed whose columns correspond to the respective joint state in the input.
         """
         get_global_link_axis = functools.partial(self.get_global_link_axis, axis=axis)
         return self.make_function(
             "a", link, get_global_link_axis, n=n, numpy_output=numpy_output
         )
+
+    @arrayify_args
+    def rnea(self, q: ArrayType, qd: ArrayType, qdd: ArrayType) -> ArrayType:
+        """! Compute inverse Dynamics with RNEA.
+        @ param  q: Joint position (unit rad).
+        @ param  qd: Joint velocity.
+        @ param  qdd: Joint accerleration.
+        """
```

Comment:
Replace documentation:

```python
"""! Compute inverse dynamics with RNEA.

NOTE, currently this method only supports revolute/continuous joints. 

@ param  q: Joint position (unit rad).
@ param  qd: Joint velocity.
@ param  qdd: Joint accerleration.
"""
```

Also, add a check at the start of the `rnea` method that makes sure only revolute/continuous joints exist in the URDF, if prismatic joints are found then an error should be thrown. This check should be as follows:

```python
# Ensure URDF only contains continuous/revolute joints
rnea_supported_joint_types = {'revolute', 'continuous'}
for joint_name, joint in self.urdf.joint_map.items():
    if joint.type not in rnea_supported_joint_types:
        raise JointTypeNotSupported(joint.type)
    
```

Note, continuous joints are revolute without limits.